src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>GHGA Data Portal</title>
+    <title>The German Human Genome-Phenome Archive</title>
```

Comment:
We're already using "GHGA Data Portal" in the title strategy in Angular since it is shorter and more specific.

You see this title quickly before Angular kicks in, so would be also less flickering in the tab.

Reply:
I just changed it to match the current data portal, but I can leave it as it was no problem.

Reply:
We decided to use a shorter title because the space on the tabs can be limited.

Reply:
What about using `The GHGA Archive`? as a compromise? 😄